pkg/cloud/aws/actuators/machine/actuator_test.go

```diff
@@ -195,38 +219,91 @@ func TestActuator(t *testing.T) {
 		name                    string
 		machine                 *clusterv1.Machine
 		error                   string
-		operation               func(actuator *Actuator, cluster *clusterv1.Cluster, machine *clusterv1.Machine)
+		operation               func(client client.Client, actuator *Actuator, cluster *clusterv1.Cluster, machine *clusterv1.Machine)
 		describeInstancesOutput *ec2.DescribeInstancesOutput
 		runInstancesErr         error
 		describeInstancesErr    error
 		terminateInstancesErr   error
 		lbErr                   error
 	}{
 		{
-			name:    "Update with success",
+			name:    "Create machine with success",
 			machine: machine,
-			operation: func(actuator *Actuator, cluster *clusterv1.Cluster, machine *clusterv1.Machine) {
+			operation: func(objectClient client.Client, actuator *Actuator, cluster *clusterv1.Cluster, machine *clusterv1.Machine) {
+				createErr := actuator.Create(context.TODO(), cluster, machine)
```

Comment:
There's no checking of `createErr`. Can this fail?

Reply:
Done. assert move right after the `Create` op call